operator/pkg/apis/istio/v1alpha1/validation/validation_test.go

```diff
@@ -12,9 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package validation
+package validation_test
```

Comment:
why is this a different package?

Reply:
Import loop, because the tests are now pulling all the profiles in. I think its pretty useful to add this test since we have had many issues in the past with this. Go treats `package_test` specially btw for this purpose